### PR TITLE
Unset width for bulkSelect in prep dialog

### LINKED ISSUE #4722: w-full class applied to dialog fields causing max expansion unnecessarily
**Describe the bug**

Fields in dialogs such as add preparations to new loans and Return Loan are expanded fully across the width of the dialog because of a class being applied here inappropriately. 

**To Reproduce**

1. Go to new loan
2. Enter cat numbers or choose record set with preparations
3. See width of bulk select in choose preps dialog
4. Save new loan and click `Return Loan` button
5. See width of Returned Date and Received By fields 

**Screenshots**

See selecting and un-selecting the `w-full` class for the Bulk Select field: 

https://github.com/specify/specify7/assets/135047322/257b9cd4-b92e-4b26-b1d8-8501e9c9edf0

- OS: macOS
- Browser: Chrome
- Specify 7 Version: `edge`
- Database Name: pripaleo
- Collection name: Paleontological
- User Name: testuser

### PR DESCRIPTION
**Pull request by CarolineDenis**
Fixes #4722

### Checklist

- [x] Self-review the PR after opening it to make sure the changes look good
      and self-explanatory (or properly documented)
- [x] Add relevant issue to release milestone

### Testing instructions

- Click on interaction
- Choose a RC 

- [ ]  See that Bulk Select is not taking full width

### CODE REVIEW

**Comment by CarolineDenis** [comment]
Screenshot 2024-04-02 at 8 16 26 AM

**Review by emenslin — APPROVED** [review]
### Testing instructions

- Click on interaction
- Choose a RC 

- [x]  See that Bulk Select is not taking full width

Bulk select does not take up the full width now! 
Screenshot 2024-04-02 110509

File: specifyweb/frontend/js_src/lib/components/Interactions/PrepDialog.tsx [hunk 1]
```diff
@@ -136,7 +136,7 @@ export function PrepDialog({
         {commonText.bulkSelect()}
         <Input.Integer
           aria-label={interactionsText.selectedAmount()}
-          className="w-[unset]"
+          className="!w-[unset]"
```
Comment (maxpatiiuk) [inline specifyweb/frontend/js_src/lib/components/Interactions/PrepDialog.tsx:139]: fyi: if we have too many issues with the need for `!`, we could explore out of the tailwind helper libraries - https://github.com/specify/specify7/issues/4328

these libs would deduplicate classes, making sure that the place where the class is set last takes precedence over the earlier usages, thus reducing/removing the need for `!` and the associated bugs...

**Review by maxpatiiuk — APPROVED** [review]

**Review by combs-a — APPROVED** [review]
### Testing instructions

- Click on interaction
- Choose a RC 

- [X]  See that Bulk Select is not taking full width

image

Looking great on Firefox!

**Review by grantfitzsimmons — APPROVED** [review]

**Pull request merged**